### PR TITLE
Removed hard coded text

### PR DESCRIPTION
**Pull request by Pratham02Komm**
Removed the hard coded text and stored them in contstants

### CODE REVIEW

File: kommunicate/src/main/java/com/applozic/mobicommons/task/executor/ExecutorAsyncTask.java [hunk 1]
```diff
@@ -55,7 +60,11 @@ public Result call() throws Exception {
                 Binder.flushPendingCommands();
             } catch (Throwable t) {
                 cancelled.set(true);
-                throw t;
+                try {
+                    throw t;
+                } catch (Throwable e) {
+                    throw new RuntimeException(e);
+                }
```
Comment (AbhijeetRanjan308) [inline kommunicate/src/main/java/com/applozic/mobicommons/task/executor/ExecutorAsyncTask.java:67]: what is the use of this code ?

File: kommunicate/src/main/java/io/kommunicate/services/KmHttpClient.java [hunk 2]
```diff
@@ -96,7 +100,7 @@ public String postData(String urlString, String contentType, String accept, Stri
         return null;
     }
 
-    public String getResponseWithException(String urlString, String contentType, String accept, Map<String, String> headers) throws Exception {
+    public String getResponseWithException(String urlString, String contentType, String accept, Map<String, String> headers) throws Throwable {
```
Comment (AbhijeetRanjan308) [inline kommunicate/src/main/java/io/kommunicate/services/KmHttpClient.java:103]: any specific reason for changing Exception with Throwable ?

File: kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/fragment/MobiComConversationFragment.java [hunk 3]
```diff
@@ -400,6 +400,37 @@ public abstract class MobiComConversationFragment extends Fragment implements Vi
...
+    private static final String TEMPLATE_MSG = "templateMessage";
+    private static final String APPLI_JSON = "application/json";
+
+    /*
+    autoSuggestionMetadata.put("KM_EVENT", "QUICKREPLY_CLICK");
+                            autoSuggestionMetadata.put("skipBot", "true");
+                            autoSuggestionMetadata.put("category", "HIDDEN");
+                            sendMessage("KM_EVENT_QUICKREPLY_CLICK", autoSuggestionMetadata, null, null, Message.ContentType.DEFAULT.getValue());
+    * */
+
```
Comment (AbhijeetRanjan308) [inline kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/fragment/MobiComConversationFragment.java:432]: What is the reason of adding this code and why it's commented ?

**Review by AbhijeetRanjan308 — CHANGES_REQUESTED** [review]

File: kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/fragment/MobiComConversationFragment.java (hunk 3 above)
Comment (Pratham02Komm) [reply kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/fragment/MobiComConversationFragment.java:432]: Removed this comment, was just added at the time of declaring constants.

File: kommunicate/src/main/java/com/applozic/mobicommons/task/executor/ExecutorAsyncTask.java (hunk 1 above)
Comment (Pratham02Komm) [reply kommunicate/src/main/java/com/applozic/mobicommons/task/executor/ExecutorAsyncTask.java]: done, please check

File: kommunicate/src/main/java/io/kommunicate/services/KmHttpClient.java (hunk 2 above)
Comment (Pratham02Komm) [reply kommunicate/src/main/java/io/kommunicate/services/KmHttpClient.java]: done, please check

**Pull request merged**